packages/orchestrator-ui-components/src/utils/resultFlattener.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQlSinglePage } from '@/types';
+
+export const pagedResultFlattener = <T>(
+    pagedResult: GraphQlSinglePage<T>,
+    fields: Array<keyof T>,
+): string => {
+    const results = pagedResult.page || [];
+    return resultFlattener(results, fields);
+};
+
+export const resultFlattener = <T>(
```

Comment:
I think this is more a mapper instead of a flattener since the keys are not showing up in the output string. Maybe name it something like `badgeMapper` or something slightly more generic.

Reply:
changed the name to getConcatenatedResult that describes what it does a bit better